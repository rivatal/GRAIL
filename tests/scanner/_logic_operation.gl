&& || ! if else for while return
