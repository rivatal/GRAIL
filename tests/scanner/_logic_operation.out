AND
OR
NOT
IF
ELSE
FOR
WHILE
RETURN
